script/changelog/parser.ts

```diff
@@ -75,7 +84,12 @@ export async function convertToChangelogFormat(
         throw new Error(`Unable to get PR from API: ${commit.prID}`)
       }
 
-      const entry = getChangelogEntry(commit, pr)
+      const collaborators = pr.collaborators
+      const externalContributors = collaborators.filter(
+        c => coreMembers.indexOf(c) === -1
```

Comment:
is:
`!(c in coreMembers)` instead of
`c => coreMembers.indexOf(c) === -1`
clearer?

Reply:
The `in` operator checks if the specified **key** is in the object (i.e. `1 in ['a', 'b', 'c']` would return `true` since `['a', 'b', 'c'][1]` exists). There’s `.includes()`, which does the same thing as `.indexOf(...) !== -1`.

Since there’s `includes`, should we open a new issue to discuss removing `indexOf` checks that test against `-1`?

Reply:
ah good points @j-f1, maybe we'll punt for a larger discussion in a separate issue